### ISSUE TITLE
3D pose alignment

### ISSUE THREAD

**Issue by matteorr** [DenisTome/Lifting-from-the-Deep-release#46]
Hi Denis, thanks for the great work. 

_Could you kindly share the aligned 3d poses or the code with which you perform the alignment?_

Also, if I correctly understand how the alignment works, you are learning matrices **e** (of dimension J x 3L) and **A** (of dimension N x J), and N rotation matrices **Ri** (that are all about the y axis only), so that you can obtain every training set pose Pi by multiplying every element of the matrix **e** by the coefficients of a row ai of **A**, addind that to the mean pose of the training data and finally rotating it by **Ri**.

Once the optimization is done you can replace every training pose Pi with its `non-rotated' version (mu + ai * **e**) which basically points in the same direction of mu. 

Could you please confirm my description is correct? If yes, could you explain why you need the noise variance sigma? If not, could you please help me understand?

Thanks a lot!

**Comment by DenisTome** [comment]
That is correct.
sigma comes from probabilistic PCA which is a design decision we made then is reflected in Eq. 4.

If you look at the advantages of PPCA vs. PCA you can see why we chose that.
